pom.xml

```diff
@@ -49,7 +49,7 @@
         <dep.accumulo.version>1.7.4</dep.accumulo.version>
         <dep.accumulo-hadoop.version>2.7.7-1</dep.accumulo-hadoop.version>
         <dep.antlr.version>4.9.3</dep.antlr.version>
-        <dep.airlift.version>214</dep.airlift.version>
+        <dep.airlift.version>215</dep.airlift.version>
```

Comment:
?

Reply:
Assuming that's the airlift version which includes https://github.com/airlift/airlift/pull/986.

This PR is blocked until newer airlift is released.